### ISSUE TITLE
Obsidian like image link support 

### ISSUE THREAD

**Issue by 1001ruchka** [GitJournal/GitJournal#350]
Please add obsidian like image link support:
![[image-name.jpg]]

**Comment by vHanda** [comment]
Closing as duplicate of https://github.com/GitJournal/GitJournal/issues/222

I'm really hoping I can get this done this month - I had been blocked by ios issues for the past week, now I'm finally unblocked, so I'll finish Multi Repo support and then get started with Obsidian Compatibility.
